cmd/collection/main.go

```diff
@@ -388,36 +384,11 @@ func main() {
 
 			signer := verification.NewSingleSigner(staking, node.Me.NodeID())
 
-			// check if required fields are left empty
-			if accessAddress == "" {
-				return nil, fmt.Errorf("flag `access-address` required")
-			}
-			if qcContractAddress == "" {
-				return nil, fmt.Errorf("flag `qc-contract-address` required")
-			}
-
-			// loads the private account info for this node from disk for use in the QCContractClient.
-			accountInfo, err := loadEpochQCPrivateData(node)
-			if err != nil {
-				return nil, err
-			}
-
-			// construct signer from private key
-			sk, err := sdkcrypto.DecodePrivateKey(accountInfo.SigningAlgorithm, accountInfo.EncodedPrivateKey)
-			if err != nil {
-				return nil, fmt.Errorf("could not decode private key from hex: %v", err)
-			}
-			txSigner := sdkcrypto.NewInMemorySigner(sk, accountInfo.HashAlgorithm)
-
-			// create QC vote client
-			flowClient, err := client.New(accessAddress, grpc.WithInsecure())
+			qcContractClient, err := createQCContractClient(node, accessAddress, qcContractAddress)
```

Comment:
conflict with #903 qc-contract-address moved , flag for qc-contract-address should be removed